src/Microsoft.Diagnostics.Monitoring.StartupHook/Exceptions/ExceptionSourceBase.cs

```diff
@@ -7,18 +7,18 @@
 namespace Microsoft.Diagnostics.Monitoring.StartupHook.Exceptions
 {
     /// <summary>
-    /// Represents a source of thrown exceptions.
+    /// Represents a source of exceptions.
     /// </summary>
     internal abstract class ExceptionSourceBase
     {
-        protected void RaiseExceptionThrown(Exception ex, DateTime timestamp, string? activityId, ActivityIdFormat format)
+        protected void RaiseException(Exception ex, DateTime timestamp, string? activityId, ActivityIdFormat format)
         {
-            ExceptionThrown?.Invoke(this, new ExceptionEventArgs(ex, timestamp, activityId, format));
+            Exception?.Invoke(this, new ExceptionEventArgs(ex, timestamp, activityId, format));
         }
 
         /// <summary>
-        /// Event that is raised each time an exception is thrown.
+        /// Event that is raised each time an exception is made available.
         /// </summary>
-        public event EventHandler<ExceptionEventArgs>? ExceptionThrown;
+        public event EventHandler<ExceptionEventArgs>? Exception;
```

Comment:
Any concern about this name-change with readability?

Reply:
Changed to ExceptionAvailable